dependencies.xml

```diff
@@ -67,10 +67,12 @@ Note all install methods after "main" take
     <numdifftools source="pip">0.9.39</numdifftools>
     <fmpy optional='True'/>
     <xmlschema source="pip"/>
-    <pyomo optional='True'>6.4</pyomo>
-    <glpk skip_check='True' optional='True'/>
-    <ipopt source="forge" skip_check='True' optional='True'/>
-    <cyipopt skip_check='True' optional='True'/>
+    <pyomo>6.4</pyomo>
+    <glpk/>
+    <ipopt/>
+    <cyipopt/>
+    <download-extensions source="pyomo"/>
+    <build-extensions source="pyomo"/>
```

Comment:
I would recommend this be called pyomo-extensions, and then if it is seen, just run both download and build.